### PR TITLE
Blog migration

### PR DESCRIPTION
**Pull request by cassidoo**

### CODE REVIEW

File: src/components/Footer.astro [hunk 1]
```diff
@@ -30,10 +30,10 @@ import logo from "../assets/contendalogo.svg";
 			<a class="hide-small f6 link white dim mr3 mr4-ns" href="#case-studies"
 				>Case Studies</a
 			>
```
Comment (tomzorz) [inline src/components/Footer.astro:32]: Is this typical formatting style for html/astro nowadays? Just the > on a separate new line, not the entire closing tag? (Not complaining, more like wondering haha)

File: src/components/Post.astro [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		bottom: 16px;
+		font-family: "Inter", sans-serif;
+		font-weight: 300;
+		font-size: 14px;
+	}
+</style>
+
+<a class="card br3 mw5 mb3 center" href={`/blog/${slug}`}>
+	<img
+		src={images[Math.floor(Math.random() * images.length)]}
```
Comment (tomzorz) [inline src/components/Post.astro:39]: future improvement: shuffle the list and use the shuffled ordering sequentially (right now sometimes the page has the same image twice in a row, looks a bit off)

File: src/layouts/BlogPost.astro [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+					{
+						tags.map((t) => {
+							return <li>{t}</li>;
+						})
+					}
+				</ul>
+			</div> -->
+			<div class="tc">
+				<a
+					class="yeehaw f4 link grow b br3 ph4 pv2 ma5 dib tc"
```
Comment (tomzorz) [inline src/layouts/BlogPost.astro:73]: `yeehaw` lmao

**Review by tomzorz — APPROVED** [review]
Awesome stuff, looks great!

**Review by rachelkitch22 — APPROVED** [review]
All comments addressed!! Looks amazing queen

**Pull request merged**